doc/source/user_information/user_reference/runtime_parameters.rst

```diff
@@ -208,6 +209,13 @@ Android Specific Runtime Parameters
 :screen_on: A ``boolean`` to specify whether the devices screen should be
     turned on. Defaults to ``True``.
 
+:unlock_screen: A ``String`` to specify how the devices screen should be
+    unlocked. Unlocking screen is disabled by default. ``vertical``, ``diagonal``
+    and ``horizontal`` are the supported values (see ``swipe_to_unlock()``).
```

Comment:
` ``swipe_to_unlock()`` ` should be replaced by the appropriate restructured text reference unless the rest of that doc uses that, e.g. :
```
:meth:`devlib.AndroidTarget.swipe_to_unlock`
```